SparkPlugs/main/default/aura/SparkPlugExampleForProcessRefund/SparkPlugExampleForProcessRefundController.js

```diff
@@ -0,0 +1,22 @@
+({
+	doInit : function(component, event, helper) {
+        let compEvent = $A.get('e.FDService:SparkPlugLoadedEvent');
+        compEvent.setParams({extensionPoint : component.get('v.extensionPoint')});
+        compEvent.fire();
+	},
+    processRefund2 : function(component, event, helper) {
+        let action = component.get('c.processRefund');
```

Comment:
let's add comments to everything. It should be 100% clear what happens in example and why.

Reply:
And example should be super minimalistic. Please remove everything which is unnesessary.